src/Bootstrap/LoadConfiguration.php

```diff
@@ -22,43 +23,8 @@ public function bootstrap(Application $app)
 
         $app->instance('config', $config = new Repository($items));
 
-        $this->loadConfigurationFiles($app, $config);
+        $this->loadConfigurationFiles($app, __DIR__.'/../../fixture/config/');
 
         mb_internal_encoding('UTF-8');
     }
-
-    /**
-     * Load the configuration items from all of the files.
-     *
-     * @param  \Illuminate\Contracts\Foundation\Application  $app
-     * @param  \Illuminate\Contracts\Config\Repository  $config
-     *
-     * @return void
-     */
-    protected function loadConfigurationFiles(Application $app, RepositoryContract $config)
```

Comment:
@GrahamCampbell 
How about?
```php
protected function loadConfigurationFiles(Application $app, RepositoryContract $config, $configPath = null)
{
    foreach ($this->getConfigurationFiles($app, $configPath) as $key => $path) {
....
```

and then for below `getConfigurationFiles`:
```php
protected function getConfigurationFiles(Application $app, $$configPath = null)
{
    $files = [];

    $path = $configPath ? realpath($configPath) : $app->configPath();
```

This would bring back the original signature(s) and add the necessary optional argument.